tests/e2e/basic/conftest.py

```diff
@@ -183,26 +183,28 @@ def create_profiles(request, testbed, get_security_flags, get_markers, instantia
     if mode not in ["BRIDGE", "NAT", "VLAN"]:
         print("Invalid Mode: ", mode)
         yield False
-    instantiate_profile.delete_profile_by_name(profile_name="Equipment-AP-" + mode)
+    instantiate_profile.delete_profile_by_name(profile_name=testbed + "-Equipment-AP-" + mode)
     for i in setup_profile_data[mode]:
         for j in setup_profile_data[mode][i]:
             instantiate_profile.delete_profile_by_name(
                 profile_name=setup_profile_data[mode][i][j]['profile_name'])
-    instantiate_profile.delete_profile_by_name(profile_name="Automation-Radius-Profile-" + mode)
+    instantiate_profile.delete_profile_by_name(profile_name=testbed + "-Automation-Radius-Profile-" + mode)
```

Comment:
Does this delete a profile on the cloud? If so then should this be made unique to the run of the pytest?

Reply:
yes its unique to each run and testbed parameter makes the uniqueness

Reply:
Can you please provide an example?